homeassistant/components/mastodon/__init__.py

```diff
@@ -81,38 +80,6 @@ async def async_unload_entry(hass: HomeAssistant, entry: MastodonConfigEntry) ->
     )
 
 
-async def async_migrate_entry(hass: HomeAssistant, entry: MastodonConfigEntry) -> bool:
```

Comment:
Why do we remove this? I don't know when it was implemented but it should be in a separate PR in that case.

Reply:
I shouldn't have removed that, got carried away with anything that said migrate.
I've added this back.